cloudDetails/aws.json

```diff
@@ -79,6 +79,9 @@
       "west-aws-image-ami": "ami-0964546d3da97e3ab",
       "local-ssd": "false",
       "aws-enable-multiple-stores": null
+    },
+    "benchArgs": {
+      "tpcc": "-a 125 -L \\'--provider-override=s3 --bucket-override=cloud-report-tpcc --auth-params-override=\"AUTH=implicit&AWS_REGION=us-east-1\"\\'"
```

Comment:
Since we will run "_75, 100, 125, 150 per vCPU_ ", I guess your idea is to manually change this value to run with a specific `warehouse_per_vcpu`, i.e. we'll need to generate 4 scripts for each machine type. 

Maybe it's easier if we pass the `f_active_per_core` though an env var from the generated script. 
E.g. We can add one line in the template in `cmd/generate.go`:
```
TPCC_WAREHOURSE_PER_VCPU=${TPCC_WAREHOURSE_PER_VCPU:=125}
```
and under `function bench_tpcc()`, we run `tpcc.sh` with 
```
run_under_tmux "tpcc" "$CLUSTER:2" "./scripts/gen/tpcc.sh -a  $TPCC_WAREHOURSE_PER_VCPU $tpcc_extra_args ${pgurls[@]}"
```
In this way we just need to generate 1 script per machine type to run with 4 different warehouse-per-vcpu values.